docs/rfcs/active/dashboards/1778-dashboards.md

```diff
@@ -0,0 +1,117 @@
+```
+urn:li:chart:(Redash,<<chart_id>>)
+```
+
+### Chart metadata
+Dashboarding tools generally have different jargon to denote a chart.
+They are called as [Look](https://docs.looker.com/exploring-data/saving-and-editing-looks) in Looker 
+and [Visualization](https://redash.io/help/user-guide/visualizations/visualization-types) in Redash.
+But, irrespective of the name, charts are the different tiles which exists in a dashboard.
+Charts are mainly used for delivering some information in a visual way to make it easily understandable.
+They might be using single or multiple data sources and generally have an associated query running against
+the underlying data source to generate the data that it will present.
+
+Below is a list of metadata which can be associated with a chart:
+
+- Title
+- Description
+- Type (Bar chart, Pie chart, Scatter plot etc.)
+- Input sources
+- Query (and its type)
+- Access level (public, private etc.)
+- Ownership
+- Status (removed or not)
+- Audit info (last modified, last refreshed)
+
+### Dashboard metadata
+Aside from containing a set of charts, dashboards carry metadata attached to them.
+Below is a list of metadata which can be associated with a dashboard:
+
+- Title
+- Description
+- List of charts
+- Access level (public, private etc.)
+- Ownership
+- Status (removed or not)
+- Audit info (last modified, last refreshed)
+
+### Metadata graph
+
+![dashboards_graph](imgs/dashboards_graph.png)
+
+An example metadata graph showing complete data lineage picture is shown above.
+In this picture, `Dash_A` and `Dash_B` are dashboards and they are connected to charts through `Contains` edges.
+`C1`, `C2`, `C3` and `C4` are charts and they are connected to underlying datasets through `DownstreamOf` edges.
+`D1`, `D2` and `D3` are datasets.
+
+## How we teach this
+
+We should create/update user guides to educate users for:
+ - Search & discovery experience (how to find a dashboard in DataHub)
+ - Lineage experience (how to find upstream datasets of a dashboard and how to find dashboards generated from a dataset)
+
+## Alternatives
+
+In this design, there is an underlying assumption that input sources for charts will be a list of datasets.
+This is slightly different from our Linkedin internal implementation in which metrics are the input sources for the charts.
+We'll not follow this for open source DataHub for below reasons: 
+ - Metric entity is not open source
+ - Popular dashboarding tools (Looker and Redash) generally use datasets as inputs directly and allow slicing & dicing 
+ on the dataset itself using their queries. In some way, they are generating the metrics on the fly.
+
+## Rollout / Adoption Strategy
+
+The design is supposed to be generic enough that any user of the DataHub should easily be able
+to onboard their dashboard metadata to DataHub irrespective of their dashboarding platform.
+
+Only thing users will need to do is to write an ETL script customized for their 
```

Comment:
Do we intent to provide any sample crawlers?

Reply:
We don't have any plans for this right now but maybe `Tableau` (if that's something we'll crawl at Linkedin going forward).